client/client.go

```diff
@@ -414,6 +414,24 @@ func (c *Client) Info() (map[string]interface{}, error) {
 	return hash, nil
 }
 
+func (c *Client) Queues() (map[string]interface{}, error) {
```

Comment:
Is this the right method name, is `QueueSizes` better? As is, it needs to return `map[string]uint64`. Should it take any arguments, e.g. like a list of queue names to filter?

Reply:
I took the name from the hash key `queues`, but queue sizes would indeed better reflect its purpose.